deploy/olm/pxc/percona-xtradb-cluster-operator.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```

Comment:
Will we drop these files and move them to the PMM once we'll have a catalog available?

Reply:
Deleted.